### PR TITLE
HPCC-14690 Add thousand separators to files in grid

### PR DESCRIPTION
**Pull request by miguelvazq**
Add thousand seperators to files and landing zone files.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

File: esp/src/eclwatch/DFUQueryWidget.js [hunk 1]
```diff
@@ -76,7 +77,7 @@ define([
 
     "hpcc/TableContainer"
 
-], function (declare, lang, i18n, nlsHPCC, arrayUtil, dom, domAttr, domConstruct, domClass, domForm, date, on, topic,
+], function (declare, lang, i18n, nlsHPCC, arrayUtil, dom, domAttr, domConstruct, domClass, domForm, date, on, topic, number,
```
Comment (GordonSmith) [inline esp/src/eclwatch/DFUQueryWidget.js]: number is not used?

**Comment by richardkchapman** [comment]
Typo in commit message (should read separators not seperators)

**Comment by richardkchapman** [comment]
@miguelvazq Back to you

**Comment by miguelvazq** [comment]
@GordonSmith Number removed.

**Comment by GordonSmith** [comment]
Looks good.

**Pull request merged**